docs/resources/script.md

```diff
@@ -0,0 +1,36 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "coder_script Resource - terraform-provider-coder"
+subcategory: ""
+description: |-
+  Use this resource to run a script from an agent.
+---
+
+# coder_script (Resource)
+
+Use this resource to run a script from an agent.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `agent_id` (String) The "id" property of a "coder_agent" resource to associate with.
+- `display_name` (String) The display name of the script to display logs in the dashboard.
+- `script` (String) The script to run.
+
+### Optional
+
+- `cron` (String) The cron schedule to run the script on. This is a cron expression.
+- `icon` (String) A URL to an icon that will display in the dashboard. View built-in icons here: https://github.com/coder/coder/tree/main/site/static/icon. Use a built-in icon with `data.coder_workspace.me.access_url + "/icon/<path>"`.
```

Comment:
Why do we have icon? Do we want to display all of them on the dashboard?

Reply:
Yes. These scripts will be displayed in the dashboard.